chew.go

```diff
@@ -151,6 +154,34 @@ func processURL(url string, ctxs ...context.Context) ([]common.Chunk, error) {
 		ctx = ctxs[0]
 	}
 
+	// if the url is a file path we can just open the file and process it directly
+	if strings.HasPrefix(url, "file://") {
+		filePath, _ := strings.CutPrefix(url, "file://")
```

Comment:
You missed my point 😅


```suggestion
	if filePath, found := strings.CutPrefix(url, "file://"); found {
```

Reply:
> You missed my point 😅

😮‍💨, my bad.